addons/evm/src/commands/actions/transfer_eth.rs

```diff
@@ -0,0 +1,381 @@
+use alloy::rpc::types::TransactionRequest;
+use std::collections::HashMap;
+use txtx_addon_kit::types::commands::{
+    CommandExecutionFutureResult, CommandExecutionResult, CommandImplementation,
+    PreCommandSpecification,
+};
+use txtx_addon_kit::types::frontend::{Actions, BlockEvent};
+use txtx_addon_kit::types::signers::{
+    SignerActionsFutureResult, SignerInstance, SignerSignFutureResult,
+};
+use txtx_addon_kit::types::stores::ValueStore;
+use txtx_addon_kit::types::{
+    commands::CommandSpecification,
+    diagnostics::Diagnostic,
+    types::{Type, Value},
+};
+use txtx_addon_kit::types::{
+    signers::SignersState, types::RunbookSupervisionContext, ConstructDid,
+};
+use txtx_addon_kit::uuid::Uuid;
+
+use crate::codec::CommonTransactionFields;
+use crate::commands::actions::check_confirmations::CheckEvmConfirmations;
+use crate::commands::actions::sign_transaction::SignEvmTransaction;
+use crate::constants::RPC_API_URL;
+use crate::rpc::EvmRpc;
+use crate::typing::EVM_ADDRESS;
+use txtx_addon_kit::constants::TX_HASH;
+
+use super::get_signer_did;
+
+lazy_static! {
+    pub static ref TRANSFER_ETH: PreCommandSpecification = define_command! {
+        TransverEth => {
+            name: "Coming soon",
+            matcher: "transfer_eth",
```

Comment:
`send_eth` would be shorter? We went with `send_stx`, Stacks side.

Reply:
Done in a1c6d0b9d304d55f4b091535b2444fae3b5ec99c